test/Confluent.Kafka.IntegrationTests/Tests/AssignOverloads.cs

```diff
@@ -28,12 +28,12 @@ public static partial class Tests
         /// <summary>
         ///     Simple test of both Consumer.Assign overloads.
         /// </summary>
-        [Theory, MemberData(nameof(KafkaParameters))]
+        [Theory, ClassData(typeof(KafkaParameters))]
         public static void AssignOverloads(string bootstrapServers, string topic)
         {
             var consumerConfig = new Dictionary<string, object>
             {
-                { "group.id", "test-consumer-group" },
+                { "group.id", "assign-overloads-cg" },
```

Comment:
For the future it is probably a good idea to use unique group ids for each invocation to have consistent tests (e.g., uuid or similar)